src/test/java/com/rocketden/main/api/GameTests.java

```diff
@@ -294,4 +304,39 @@ public void submitSolutionSuccess() throws Exception {
         assertEquals(submissionDto.getLanguage(), player.getLanguage());
         assertTrue(player.getSolved());
     }
+
+    @Test
+    public void sendNotificationSuccess() throws Exception {
+        UserDto host = new UserDto();
+        host.setNickname(NICKNAME);
+        host.setUserId(USER_ID);
+
+        RoomDto roomDto = RoomTestMethods.setUpRoomWithOneUser(this.mockMvc, host);
+        startGameHelper(roomDto, host);
+
+        /**
+         * TODO: If time is replaced with LocalDateTime.now(), 400 error,
+         * as it cannot convert the time field into a JSON string. We could 
+         * simply not have this field be set, and then create it on the
+         * backend, but if this could be figured out that'd be great.
+         */
+
```

Comment:
Does the `toObjectLocalDateTime` not work in this scenario?

Reply:
No it doesn't, because that converts a JSON string to an object, while this converts an object to a JSON string. I've tried putting the same Gson setup in the `toObjectLocalDateTime`, but it didn't seem to fix the issue -- so I'm stuck here. Any advice?

Reply:
So right now it's just failing in the tests right?

Reply:
If so I'd be happy with removing the comment/ignoring it

Reply:
Ok -- I think I'll update the comment to be one line and not have a TODO, but still indicate that if time is replaced with `LocalDateTime.now()`, that gets a 400 error -- it'd be good to note that for the future.

Reply:
Actually, it was failing in Postman too, so I've restructured this code and added a `GameNotificationRequest` that does not include the `time` variable; then, I set the `time = LocalDateTime.now()` on the backend. This is a fairly significant change, so although I think it looks good, I've re-requested review -- so lmk!